int32 object_id
geometry_msgs/Point32 root_point
geometry_msgs/Point32 width_height

